treeherder/webapp/api/runnable_jobs.py

```diff
@@ -19,16 +21,17 @@ def list(self, request, project):
         """
         if 'decisionTaskID' in request.query_params and len(request.query_params['decisionTaskID']):
```

Comment:
At a glance it would seem this could be written more concisely as:
```python
decision_task_id = request.query_params.get('decisionTaskID')
if decision_task_id:
    # ...
```
...unless I'm missing something? :-)
(Note in Python the empty string and similar are considered Falsey)